types/googlemaps/test/reference/geocode-tests.ts

```diff
@@ -0,0 +1,10 @@
+import Geocoder = google.maps.Geocoder;
+
+const geocoder = new Geocoder();
+
+geocoder.geocode({}, (results, status) => {
+    results; // $ExpectType GeocoderResult[]
+    status; // $ExpectType GeocoderStatus
+});
+
+export {};
```

Comment:
```suggestion
```
IMO this module exports nothing, just consumes

Reply:
There is no `import` or `export` in the file ([namespace imports](https://www.typescriptlang.org/docs/handbook/namespaces.html#aliases) are not counted). If I don't add `export {}`, then TypeScript will store all variables declared here and treat them as global. The rest of the test files use a similar technique.

Reply:
but this is a test file, for compiler related checks, it doesn't get published (as type file that is) nor being consumed at all at any stage

Reply:
oh, I see, that's becuase of global exports in the DT details, so in tests aliases are used. Sry!

Reply:
<img width="1282" alt="example1" src="https://user-images.githubusercontent.com/10235949/97118339-2e62a900-1712-11eb-8da5-be70bfd9ca19.png">
<img width="1282" alt="example2" src="https://user-images.githubusercontent.com/10235949/97118347-33275d00-1712-11eb-989d-7726c7664ee5.png">